app/views/users/comments.html.erb

```diff
@@ -1,12 +1,52 @@
-<h1> <%= link_to "#{user.name}'s", user %> Comments</h1>
-<ul>
-  <% user.comments.each do |comment| %>
-    <li>
-      <%= comment.text %> posted on <%= comment.created_at.strftime("%B %d, %Y at %I:%M %p") %>
-      <div>
-        <%= link_to "Article", comment.article %>
-      </div>
-    </li>
-  <% end %>
-</ul>
+<tr style="height:10px"></tr>
+<tr>
+  <td>
+    <table border="0" cellpadding="0" cellspacing="0">
+      <tbody>
+        <% current_page = params[:page].present? ? params[:page].to_i : 1 %>
+        <% @comments.each_with_index do |comment, index| %>
+          <tbody>
```

Comment:
I see the start of 2 `tbody` blocks here, but only 1 ending so something seems off here.